### DISCUSSION TITLE
Make the Media Carousel widget work inside Loop Grid

### DISCUSSION THREAD

**Discussion by pzh20** [elementor/elementor#23559] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

As more and more sites become dynamic, a widget like this should be able to take it's images from posts inside the Loop

### Describe the solution you'd like

Same output as is now, but with the images being taken from the current Loop Grid.

### Describe alternatives you've considered

I've searched many addons but none seem to provide a similar output

**Comment by nicholaszein** [comment]
This Request was mentioned here:

Issues:

- #24003

**Comment by m0n0mind** [comment]
Even though you might not rate this as a bug it behaves/feels like this (scaling images to crazy widths and such).
Furthermore, there seems no official documentation about which widgets are supported inside a loop grid.
Current state feels more like experimental/beta.

Therefore, I'd like to see this feature added to Elementor (Pro) core. Thanks!

**Comment by rcwd** [comment]
This problem is still unresolved!
